common/persistence/persistence_interface.go

```diff
@@ -78,6 +78,7 @@ type (
 		ListTaskQueueUserDataEntries(ctx context.Context, request *ListTaskQueueUserDataEntriesRequest) (*InternalListTaskQueueUserDataEntriesResponse, error)
 		GetTaskQueuesByBuildId(ctx context.Context, request *GetTaskQueuesByBuildIdRequest) ([]string, error)
 		CountTaskQueuesByBuildId(ctx context.Context, request *CountTaskQueuesByBuildIdRequest) (int, error)
+		CountTasksFromTaskQueue(ctx context.Context, request *CountTasksFromTaskQueueRequest) (int, error)
```

Comment:
should it be `CountTasks` or `GetTaskCount`? @dnr wdyt?

Reply:
after discussion, have changed the name to be `CountTasksExact` but would love your opinion as well, David.